wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -298,6 +298,7 @@ public void load(Device device) throws ModelException, IOException {
                                                 Adapter.newInstance(
                                                         this,
                                                         adapterName,
+                                                        null,
```

Comment:
I think by default we should set alias to the name, not null. We basically have that now with the `getAlias` call, but i'd prefer we set the alias explicitly to the name rather than the condition check there.

That way we can also avoid passing alias as an argument to unregisterAdapter in ModelInfo